tools/challenge-parser/package.json

```
@@ -1,14 +1,25 @@
 {
   "name": "challenge-parser",
```

```suggestion
  "name": "@freecodecamp/challenge-parser",
```
Is that the rename you wanted?  If so, yep, I'm totally onboard with that.

---

I happened across https://github.blog/2021-02-12-avoiding-npm-substitution-attacks/ today, so I see the need!

---

This must have been a happy accident in that case.